runtime/runtime-params-estimator/src/testbed_runners.rs

```diff
@@ -154,7 +154,7 @@ fn end_count_instructions() -> u64 {
 }
 
 fn start_count_time() -> Consumed {
-    Consumed::Instant(Instant::now())
+    Consumed::Instant(Instant::system_time())
```

Comment:
What's the point of this change? We do not need system time here, we need a way to count monotonic ticks between two moments. 
`Instant::now` does exactly what we need, i.e. 
```
A measurement of a monotonically nondecreasing clock. Opaque and useful only with Duration.
```

While  description of SystemTime 
```
A measurement of the system clock, useful for talking to external entities like the file system or other processes.
```
is not behavior we're looking for here.

Reply:
@olonho does my comment resolve your doubts?